### PR TITLE
Update batch_driver.py

### PR DESCRIPTION
**Pull request by santiagxf**
# Description

This PR introduces handling of models without a signature in the MLflow scoring script for batch

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/endpoints/batch/deploy-models/heart-classifier-mlflow/code/batch_driver.py [hunk 1]
```diff
@@ -19,18 +20,23 @@ def init():
...
+                    model.metadata.signature.inputs.pandas_types(),
+                )
             )
+        if model.metadata.signature.outputs:
+            if model.metadata.signature.outputs.has_input_names():
+                model_output_names = model.metadata.signature.outputs.input_names()
+            elif len(model.metadata.signature.outputs.input_names()) == 1:
+                model_output_names = ["prediction"]
+    else:
+        logging.warn(
```
Comment (SagarikaKengunte) [inline sdk/python/endpoints/batch/deploy-models/heart-classifier-mlflow/code/batch_driver.py:37]: Should this warning be logged in the else part of 'model.metadata.signature.inputs'?

File: cli/endpoints/batch/deploy-models/heart-classifier-mlflow/deployment-custom/code/batch_driver.py [hunk 2]
```diff
@@ -19,18 +20,23 @@ def init():
...
+                    model.metadata.signature.inputs.input_names(),
+                    model.metadata.signature.inputs.pandas_types(),
+                )
             )
+        if model.metadata.signature.outputs:
+            if model.metadata.signature.outputs.has_input_names():
+                model_output_names = model.metadata.signature.outputs.input_names()
+            elif len(model.metadata.signature.outputs.input_names()) == 1:
+                model_output_names = ["prediction"]
+    else:
```
Comment (SagarikaKengunte) [inline cli/endpoints/batch/deploy-models/heart-classifier-mlflow/deployment-custom/code/batch_driver.py:36]: Same comment as below. Shouldn't we log this warning in the else part of 'model.metadata.signature.inputs'?

File: sdk/python/endpoints/batch/deploy-models/heart-classifier-mlflow/code/batch_driver.py (hunk 1 above)
Comment (santiagxf) [reply sdk/python/endpoints/batch/deploy-models/heart-classifier-mlflow/code/batch_driver.py]: Update!

File: cli/endpoints/batch/deploy-models/heart-classifier-mlflow/deployment-custom/code/batch_driver.py (hunk 2 above)
Comment (santiagxf) [reply cli/endpoints/batch/deploy-models/heart-classifier-mlflow/deployment-custom/code/batch_driver.py:36]: Updated!

**Review by SagarikaKengunte — APPROVED** [review]

File: cli/endpoints/batch/deploy-models/heart-classifier-mlflow/deployment-custom/code/batch_driver_parquet.py [hunk 3]
```diff
@@ -1,7 +1,11 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license.
+
 import os
 import glob
 import mlflow
 import pandas as pd
+import logging
```
Comment (mastloui-msft) [inline cli/endpoints/batch/deploy-models/heart-classifier-mlflow/deployment-custom/code/batch_driver_parquet.py:8]: system lib imports should come before package imports :D

**Review by mastloui-msft — APPROVED** [review]

File: cli/endpoints/batch/deploy-models/heart-classifier-mlflow/deployment-custom/code/batch_driver_parquet.py (hunk 3 above)
Comment (santiagxf) [reply cli/endpoints/batch/deploy-models/heart-classifier-mlflow/deployment-custom/code/batch_driver_parquet.py:8]: Indeed!

**Pull request merged**